e2e/libs/jest-reporter.js

```diff
@@ -0,0 +1,80 @@
+const fs = require('fs-extra');
+const path = require('path');
+const { fileLogger } = require('./logger');
+const winston = require("winston");
+
+module.exports = class JestReporter {
+
+  constructor(globalConfig, options) {
+    if (globalConfig.verbose === true) {
+      throw Error("Verbose must be false or Console messages won't be available.")
+    }
+    this._globalConfig = globalConfig;
+    this._options = options;
+    this.logDir = this._options.outputdir || 'logs/jest';
+    this.summaryFile = this._options.filename || 'test-results-summary.json';
+  }
+
+  onTestStart(test) {
+    const time = new Date().toLocaleTimeString();
+    console.info(`Running ${path.parse(test.path).name} at ${time}`);
+  }
+
+  // @ts-ignore
```

Comment:
tslint is checking all files (.ts, .js) because `tsconfig.json` is config with `"inclue": ["/*"]`. tslint is soon to be replaced by eslint. We can safely ignore this "bug" at this time.